src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -1716,7 +1716,7 @@ til::point Terminal::GetViewportRelativeCursorPosition() const noexcept
 // These functions are used by TerminalInput, which must build in conhost
 // against OneCore compatible signatures. See the definitions in
 // VtApiRedirection.hpp (which we cannot include cross-project.)
-// Since we do nto run on OneCore, we can dispense with the compatibility

```

Comment:
Not sure I capture the intent of the original comment. The spell checker complains about nto and the only way I got the sentence to work was like in my change.

Reply:
Should go into the spell check commit